src/NetAnalyzers/Core/AnalyzerReleases.Unshipped.md

```diff
@@ -4,6 +4,11 @@ Rule ID | Category | Severity | Notes
 --------|----------|----------|-------
 CA1418 | Interoperability | Warning | UseValidPlatformString, [Documentation](https://docs.microsoft.com/dotnet/fundamentals/code-analysis/quality-rules/ca1418)
 
+### New Rules
```

Comment:
Not sure if it supposed to create new table for new Rules, it is used to be one table and appending the new rules, is this generated or did you added this manually? cc @mavasani

Reply:
Good catch!
I think the analyzer should even warn if this was manually edited.